qiskit_nature/properties/second_quantization/electronic/electronic_density.py

```diff
@@ -0,0 +1,209 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The ElectronicDensity property."""
+
+from __future__ import annotations
+
+from itertools import permutations, product
+
+import re
+import logging
+
+import numpy as np
+
+from qiskit_nature import ListOrDictType, QiskitNatureError, settings
+from qiskit_nature.operators.second_quantization import FermionicOp
+from qiskit_nature.results import EigenstateResult
+from .bases import ElectronicBasis, ElectronicBasisTransform
+from .integrals import (
+    ElectronicIntegrals,
+    IntegralProperty,
+    OneBodyElectronicIntegrals,
+    TwoBodyElectronicIntegrals,
+)
+from .particle_number import ParticleNumber
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ElectronicDensity(IntegralProperty):
```

Comment:
Was this class added in error? The existing ones in this folder are being deprecated right.

Reply:
Oh yes, this was done by mistake will migrating over from an earlier version I had offline :see_no_evil: